packages/coreutils/package.json

```diff
@@ -64,5 +63,6 @@
   },
   "publishConfig": {
     "access": "public"
-  }
+  },
+  "module-node": "dist/index.node.es6"
```

Comment:
Could we keep `module-node` close to the other output configuration to ease configuration maintenance?

Reply:
I didn't do any of the sorting manually, but something did it automatically at some point. I can't seem to get it to reproduce this though. Maybe this was an old config of the linter that was pulled in automatically by git when rebasing some old code? 🤔  If the linters are no longer set up to sort things, I will undo the sorting.